src/lib.rs

```diff
@@ -53,20 +53,42 @@ pub struct Entry {
     pub bcd_revision: u8,
 }
 
-/// Failure type for trying to find the SMBIOS `Entry` structure in memory.
+/// The version number associated with the Smbios `EntryPoint`
+#[derive(Copy, Clone, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
+pub struct SmbiosVersion {
+    pub major: u8,
+    pub minor: u8,
+}
+
+impl From<(usize, usize)> for SmbiosVersion {
+    fn from(other: (usize, usize)) -> SmbiosVersion {
+        SmbiosVersion {
+            major: other.0 as u8,
+            minor: other.1 as u8,
+        }
+    }
+}
+
+#[derive(Copy, Clone, Debug, Eq, Hash, PartialEq)]
+pub(crate) struct SmbiosBound {
```

Comment:
Is it actually possible for a user to use one of these? They can't read the on in `Structures` or call `to_bound`